File: 3194-find-words-containing-character/find-words-containing-character.java
class Solution {
    public List<Integer> findWordsContaining(String[] words, char x) {
        List<Integer> ans=new ArrayList<>();
        String temp=new String();
        for(int i=0;i<words.length;i++){
            temp=words[i];
            for(char j:temp.toCharArray()){
                if(j==x){
                    ans.add(i);
                    break;
                }
            }
        }
        return ans;
    }
}

File: 3194-find-words-containing-character/find-words-containing-character.cpp
class Solution {
public:
    vector<int> findWordsContaining(vector<string>& words, char x) {
        vector<int>ans;
        int n=words.size();
        string temp;
        for(int i=0;i<n;i++){
            temp=words[i];
            for(char &j:temp){
                if(j==x){
                    ans.push_back(i);
                    break;
                }
            }
        }
        return ans;
    }
};